contrib/platform/test/com/sun/jna/platform/win32/PsapiTest.java

```diff
@@ -248,4 +248,22 @@ public void testGetPerformanceInfo() {
         assertTrue(Psapi.INSTANCE.GetPerformanceInfo(perfInfo, perfInfo.size()));
         assertTrue(perfInfo.ProcessCount.intValue() > 0);
     }
+
+    @Test
+    public void testEnumProcesses() {
+        final int DWORD_SIZE = 4;
+        int size = 128 * DWORD_SIZE;
```

Comment:
The code works but I'm confused what you are intending your `size` variable to represent.  Here it looks like you want the size in bytes, but you also use it later to declare the size of the `int[]` array.

Reply:
Well spotted. This part was lost in the translation from some old C code to Java.
I'll provide a fixed version when I've created the utility and test ran it on Windows again, hopefully later today.

Reply:
Also (my opinion, feel free to disagree) I think incrementing by a fixed "sane" amount in the hundreds would be better than doubling each iteration.

Reply:
That would work too, I do not have any big opinion on this.
What is a good number of tasks to start with and how much should I increment on each retry?

Reply:
You might consult `java.util.Random` for an answer. :)

More seriously... my experience has been between 100-150 processes on my personal machine and on the order of 500 or so on my work machine.  I'd try to avoid allocating more than a page of memory at once (generally 4096 bytes, so 1024 processes).  So that should bound it reasonably well. 

If I were writing it I'd probably go with 256, 512, or 1024, depending on my caffeine level.